website/src/templates/docs.js

```diff
@@ -303,7 +303,7 @@ const Content = props => (
         boxShadow: '-4px 0 0 rgba(9, 30, 66, 0.09)',
         boxSizing: 'border-box',
         fontFamily: 'Consolas,Menlo,Monaco,"Andale Mono","Ubuntu Mono",monospace',
-        lineHeight: '1.4',
+        lineHeight: '1.2',
```

Comment:
Looks like the difference is that `prism-react-renderer` uses block elements for lines of code but `@mapbox/rehype-prism` uses inline elements

The lineHeight change can be reverted, I changed it so this PR would be visually similar to the current site